src/Page/Submission/Team3/Team 3/javascripts/main.js

```diff
@@ -0,0 +1,54 @@
+/* App.js */
+var React = require('react');
+var Component = React.Component;
+var CanvasJSReact = require('./canvasjs.react');
+var CanvasJS = CanvasJSReact.CanvasJS;
+var CanvasJSChart = CanvasJSReact.CanvasJSChart;
+class App extends Component {
+	render() {
+		const options = {

```

Comment:
Why don't you add this part to the Team3.js file? Your main.js is not imported anywhere. It cannot be rendered. Also, you can have your html file instead the rendering function.